tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -1885,27 +1886,272 @@ TEST (nnstreamer_capi_util, info_set_tname_1)
  */
 TEST (nnstreamer_capi_util, info_get_tname_n)
 {
-  int status = ml_tensors_info_get_tensor_name (nullptr, 0, nullptr);
+  int status;
+  ml_tensors_info_h info;
+  char *name = NULL;
+
+  status = ml_tensors_info_create (&info);
+  ASSERT_EQ (status, ML_ERROR_NONE);
+  status = ml_tensors_info_set_count (info, 1);
+  ASSERT_EQ (status, ML_ERROR_NONE);
+
+  status = ml_tensors_info_get_tensor_name (nullptr, 0, &name);
   ASSERT_EQ (status, ML_ERROR_INVALID_PARAMETER);
+  status = ml_tensors_info_get_tensor_name (info, 0, nullptr);
+  ASSERT_EQ (status, ML_ERROR_INVALID_PARAMETER);
+  status = ml_tensors_info_get_tensor_name (info, 2, &name);
+  ASSERT_EQ (status, ML_ERROR_INVALID_PARAMETER);
```

Comment:
How about using `EXPECT_EQ()` instead of `ASSERT_EQ()` for line 1898-1903 which this unittest really wants to check.

Reply:
updated

Reply:
There are a lot of continuous ASSERT tests, I think SVACE will make a problem.